neural_compressor/compression/callbacks.py

```diff
@@ -213,22 +214,28 @@ def model(self, user_model):
         if user_model is None:
             return
 
-        if self.cfg.model.framework == 'NA':
-            self.framework = get_model_fwk_name(
-                user_model.model if isinstance(user_model, BaseModel) else user_model)
-            if self.framework == "tensorflow":
+        self.framework = get_model_fwk_name(
+            user_model.model if isinstance(user_model, BaseModel) else user_model)
+        if self.framework == "tensorflow":
+            try:
                 if self.cfg.quantization.approach == "quant_aware_training":
                     self.framework = 'tensorflow_itex'
                 else:
                     from ..model.tensorflow_model import get_model_type
-                    if get_model_type(user_model) == 'keras' and self.cfg.model.backend == 'itex':
+                    if get_model_type(user_model) == 'keras' and self.cfg.quantization.backend == 'itex':
                         self.framework = 'keras'
-            if self.framework == "pytorch":
-                if self.cfg.model.backend == "default":
+            except Exception as e:
+                pass
+
+        if self.framework == "pytorch":
+            try:
+                if self.cfg.quantization.backend == "default":
                     self.framework = "pytorch_fx"
-                elif self.cfg.model.backend == "ipex":
+                elif self.cfg.quantization.backend == "ipex":
```

Comment:
coding style inconsistent. if you take a look above, it's self.conf but not self.cfg